### ISSUE TITLE
Completion handler not always called when fetching messages in collection

### ISSUE THREAD

**Issue by antoinelamy** [sendbird/sendbird-syncmanager-ios#5]
When calling the fetch function, it seems to works but sometimes the completion handler never gets called and from that time, every subsequent call to fetch won't call the completion handler either. I am sure the collection is not nil and get called.

```swift
print("Calling fetch on \(String(describing: messageCollection))")
self.messageCollection?.fetch(in: SBSMMessageDirection.previous, completionHandler: { [weak self] (error) in
    print("Completion handler called")
})
```

That code produce the following in the console when entering / leaving the chat screen:
```
Calling fetch on Optional(<SBSMMessageCollection: 0x60000324d8c0>)
Completion handler called
Calling fetch on Optional(<SBSMMessageCollection: 0x600003249200>)
Calling fetch on Optional(<SBSMMessageCollection: 0x600003249200>)
Calling fetch on Optional(<SBSMMessageCollection: 0x60000324ddd0>)
```

**Comment by biolattof** [comment]
Keep having the same problema at v1.1.6

**Comment by janna-sendbird** [comment]
@antoinelamy and @biolattof Thank you both for reporting in on this issue. This should be resolved in the latest version v1.1.7. Please try it out and let us know if you still have any issues by opening a support ticket on your SendBird app dashboard to support.
